src/components/common/TmBalance.vue

```diff
@@ -538,6 +552,7 @@ export default {
           /* istanbul ignore next */
           this.$apollo.queries.overview.refetch()
           this.$apollo.queries.balances.refetch()
+          this.$apollo.queries.rewards.refetch()
```

Comment:
are we batching these requests @faboweb ?

Reply:
I think so, would need to check myself